### PR TITLE
Add copy about featuring job in newsletter

### PR DESCRIPTION
**Pull request by matyikriszta**
## Description
This PR adds a paragraph to the job#new page about featuring the job in our bimonthly (once every two months) newsletter. @KimberleyCook please feel free to suggest copy changes. Additionally, let me know if you think we should add this to any other pages?

## Status
Ready for Review

## Related Trello ticket
https://trello.com/c/qIkc57WW

## Screenshots
Screenshot_2020-08-11 codebar

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -366,6 +366,7 @@ es:
         salary: The job pays an appropriate salary
         short: "It's short and to the point"
         approval_info: "*Once submitted, a job will be visible once it has been reviewed and approved."
+        newsletter: "For an additional <strong>£250</strong> you can list the job in our newsletter. Our newsletter goes out every other month to an audience of 10000 students and coaches. To learn more about including your job in our newsletter send us an email at <a href='mailto:hello@codebar.io'>hello@codebar.io</a>."
```
Comment (despo) [inline config/locales/es.yml:369]: Can we tweak this text to
> newsletter: "For an additional <strong>£250</strong> you can have your job listing featured in our bimonthly newsletter and made available to an audience of over 10000 students and coaches. To find out more send us an email at <a href='mailto:hello@codebar.io'>hello@codebar.io</a>."

Comment (matyikriszta) [reply config/locales/es.yml:369]: @despo so Ben and I had a discussion about this and bimonthly is very ambiguous. Is it twice a month or every other month? Happy to update the copy but I don't think we should use that word specifically.

Comment (despo) [reply config/locales/es.yml:369]: I don't even know why I used it, I think I was thinking of the PR's description - as I feel the same way about it, to me it means twice a month lol. Let me rephrase

Comment (matyikriszta) [reply config/locales/es.yml:369]: @despo haha, yeah, I realized I used it in the PR description while arguing against it here 🤦‍♀️

Comment (despo) [reply config/locales/es.yml:369]: Is frequency important enough to mention here? Is it not more important when the next newsletter is due, as someone wanting to list a job eg at the beginning of June is not likely to be interested to be featured in a newsletter going out in three weeks. So, can we just drop the mention?

> newsletter: "For an additional £250 you can have your job listing featured in our newsletter and made available to an audience of over 10000 students and coaches. To find out more details and when our next newsletter is due send us an email at hello@codebar.io."

Comment (matyikriszta) [reply config/locales/es.yml:369]: @despo yep, sounds good 👍

Comment (matyikriszta) [reply config/locales/es.yml]: @despo updated the copy if you want to take another look
